### PR TITLE
Update Docker Image

### PR DESCRIPTION
**Pull request by acgxv**

### CODE REVIEW

**Comment by acgxv** [comment]
@fewensa please review.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -17,44 +17,37 @@ jobs:
...
-            darwinia-collator -I pzstd
-
-          mv target/x86_64-unknown-linux-gnu/release/wbuild/crab-parachain-runtime/*.wasm shared/crab-parachain-runtime
-          mv target/x86_64-unknown-linux-gnu/release/wbuild/darwinia-parachain-runtime/*.wasm shared/darwinia-parachain-runtime
-
       - name: Upload shared
         uses: actions/upload-artifact@v2
         with:
-          name: darwinia-artifact
+          name: bridger-artifact
```
Comment (fewensa) [inline .github/workflows/release.yml]:
```suggestion
          name: darwinia-artifact
```

darwinia-artifact is better than bridger in this repository and the download shared is used name darwinia-artifact

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -17,44 +17,37 @@ jobs:
...
+            shared/darwinia-collator-x86_64-linux-gnu.tar.bz2 \
+            -C \
+            target/x86_64-unknown-linux-gnu/release \
             darwinia-collator
-          tar cf shared/darwinia-collator-x86_64-linux-gnu.tar.zst \
-            -C target/x86_64-unknown-linux-gnu/release \
-            darwinia-collator -I pzstd
-
-          mv target/x86_64-unknown-linux-gnu/release/wbuild/crab-parachain-runtime/*.wasm shared/crab-parachain-runtime
-          mv target/x86_64-unknown-linux-gnu/release/wbuild/darwinia-parachain-runtime/*.wasm shared/darwinia-parachain-runtime
```
Comment (fewensa) [inline .github/workflows/release.yml:52]: Keep move wasm file there, this is required for next step

**Review by fewensa — APPROVED** [review]

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -17,40 +17,40 @@ jobs:
...
+            -C \
+            target/x86_64-unknown-linux-gnu/release \
             darwinia-collator
           tar cf shared/darwinia-collator-x86_64-linux-gnu.tar.zst \
-            -C target/x86_64-unknown-linux-gnu/release \
+            -C \
+            target/x86_64-unknown-linux-gnu/release \
             darwinia-collator -I pzstd
-
           mv target/x86_64-unknown-linux-gnu/release/wbuild/crab-parachain-runtime/*.wasm shared/crab-parachain-runtime
```
Comment (acgxv) [inline .github/workflows/release.yml]: here

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -17,40 +17,40 @@ jobs:
...
           mv target/x86_64-unknown-linux-gnu/release/wbuild/darwinia-parachain-runtime/*.wasm shared/darwinia-parachain-runtime
```
Comment (acgxv) [inline .github/workflows/release.yml]: here

**Pull request merged**